v0/destinations/autopilot/transform.js

```diff
@@ -27,6 +27,9 @@ function responseBuilder(payload, message, autoPilotConfig) {
       break;
     case EventType.TRACK:
       requestConfig = defaultPostRequestConfig;
+      if (!message.context.traits.email) {
```

Comment:
I think we should give a check of whether message.context.traits exist other wise "error": "Cannot read property 'email' of undefined" will be thrown